docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,87 @@
+Run the workflow using example data
+===================================
+
+In this tutorial, you will run the workflow using example reference data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`setup`. These instructions will install all of the software you need to complete this tutorial and others.
+
+Setup
+-----
+
+1. Activate the ``nextstrain`` conda environment:
+
+   .. code:: text
+
+      conda activate nextstrain
+
+2. Change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+3. Download the example tutorial repository into a new directory ``ncov-tutorial/``:
+
+   .. code:: text
+
+      git clone https://github.com/nextstrain/ncov-tutorial
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a configuration file provided in the tutorial directory:
+
+.. code:: text
+
+   nextstrain build . --cores all --configfile ncov-tutorial/example-data.yaml
```

Comment:
This sounds like a CLI issue, but I haven't seen it myself... what's your `nextstrain --version`?

Reply:
```
$ nextstrain --version
nextstrain.cli 3.2.1
```

Installed into a fresh conda environment yesterday following the instructions in our docs

Reply:
I tested on a fresh Ubuntu VM without docker installed, and can't reproduce this. @tsibley any idea?

<details>
<summary>For the record, this is my output:</summary>

```
$ nextstrain check-setup
nextstrain-cli is up to date!

Testing your setup…

# docker is not supported
✘ no: docker is installed
✘ no: docker run works
? unknown: containers have access to >2 GiB of memory
✔ yes: image is new enough for this CLI version

# native is not supported
✘ no: snakemake is installed
✘ no: augur is installed
✘ no: auspice is installed

# aws-batch is not supported
✘ no: job description "nextstrain-job" exists
✘ no: job queue "nextstrain-job-queue" exists
✘ no: S3 bucket "nextstrain-jobs" exists

No good.  No support for any Nextstrain environment.
```

</details>

Reply:
What about if you install augur, auspice etc via conda so that there is support for the native runtime? Does the command then work without needing `--native`?

Reply:
@jameshadfield Looks like maybe you forgot `--set-default` when you ran:

```
$ nextstrain check-setup
...
All good!  Supported Nextstrain environments: native
```

The default runtime is Docker if none is set, so that's why your subsequent `nextstrain build` used it.

Reply:
Ohh ok. Somehow I missed that the error was from `nextstrain build`.

Reply:
https://github.com/nextstrain/cli/issues/164